webservices/resources/reports.py

```diff
@@ -105,11 +134,18 @@ def build_query(self, committee_id=None, committee_type=None, **kwargs):
         if kwargs.get('is_amended') is not None:
             query = query.filter(reports_class.is_amended == kwargs['is_amended'])
 
+        query = filters.filter_range(query, kwargs, self.filter_range_fields)
         return query, reports_class, reports_schema
 
     def _resolve_committee_type(self, committee_id=None, committee_type=None, **kwargs):
-        if committee_id is not None:
-            query = models.CommitteeHistory.query.filter_by(committee_id=committee_id)
+        #if kwargs.get('committee_id') is not None and len(kwargs.get('committee_id')) > 0:
+        if committee_id is not None or kwargs.get('committee_ids'):
+            comm_id = (
+                committee_id
+                if committee_id is not None
+                else kwargs.get('committee_ids')[0]
```

Comment:
@noahmanger We can't currently do bunch of committee_ids if they are different types.

So if someone wanted the reports for their house, senate and presidential race all at once, we can't currently deal with that because they are in 3 different tables and use 3 different schemas. In this proposal, we would only get one.

Our options are:
    a) don't accept multiple committee_ids 
    b) accept multiple ids only when they come from the same table.
    b) re-write the underlying data view and resource 

In my opinion, re-writing this now is more work than it is worth. Feel free to push back. I can see how this would be a useful feature.